src/utils/num.ts

```diff
@@ -97,28 +168,55 @@ export function assertInRange(
 
 /**
  * Convert BigNumberish array to decimal string array
+ * @param rawCalldata Array of BigNumberish values
  * @returns format: decimal string array
+ * @example
+ * ```typescript
+ * const bigNumberishArray: BigNumberish[] = [123, "456", 789n];
+ * const result = bigNumberishArrayToDecimalStringArray(bigNumberishArray);
+ * // result = ["123", "456", "789"]
+ * ```
  */
 export function bigNumberishArrayToDecimalStringArray(rawCalldata: BigNumberish[]): string[] {
   return rawCalldata.map((x) => toBigInt(x).toString(10));
 }
 
 /**
  * Convert BigNumberish array to hexadecimal string array
+ * @param rawCalldata Array of BigNumberish values
  * @returns format: hex-string array
+ * @example
+ * ```typescript
+ * const bigNumberishArray: BigNumberish[] = [123, "456", 789n];
+ * const result = bigNumberishArrayToHexadecimalStringArray(bigNumberishArray);
+ * // result = ["0x7b", "0x1c8", "0x315"]
+ * ```
  */
 export function bigNumberishArrayToHexadecimalStringArray(rawCalldata: BigNumberish[]): string[] {
   return rawCalldata.map((x) => toHex(x));
 }
 
 /**
  * Test if string is whole number (0, 1, 2, 3...)
+ * @example
```

Comment:
Check if @param is present in all functions (including of course at least one parameter)

Reply:
I have added @param to all function where at least one parameter exist